site/src/sphinx/advanced-retry.rst

```diff
@@ -1,8 +1,277 @@
+.. _decorator: client-decorator.html
+.. _RetryingClient: apidocs/index.html?com/linecorp/armeria/client/retry/RetryingClient.html
+.. _RetryingHttpClient: apidocs/index.html?com/linecorp/armeria/client/retry/RetryingHttpClient.html
+.. _RetryingRpcClient: apidocs/index.html?com/linecorp/armeria/client/retry/RetryingRpcClient.html
+.. _ClientBuilder: apidocs/index.html?com/linecorp/armeria/client/ClientBuilder.html
+.. _RetryStrategy: apidocs/index.html?com/linecorp/armeria/client/retry/RetryStrategy.html
+.. _Backoff: apidocs/index.html?com/linecorp/armeria/client/retry/Backoff.html
 .. _com.linecorp.armeria.client.retry: apidocs/index.html?com/linecorp/armeria/client/retry/package-summary.html
+.. _LoggingClient: apidocs/index.html?com/linecorp/armeria/client/logging/LoggingClient.html
+.. _ResponseTimeoutException: apidocs/index.html?com/linecorp/armeria/client/ResponseTimeoutException.html
 
 .. _advanced-retry:
 
 Automatic retry
 ===============
 
-TBW - See `com.linecorp.armeria.client.retry`_.
+When a client gets an error response, the client might want to retry the request depending on the response.
+You can do this by writing your own decorator_ or use one of the following out-of-the-box decorator
```

Comment:
Perhaps it's more natural as something like `This can be accomplished using a decorator_, and armeria provides the following implementations out-of-the-box.`